res/openvpn-gui-res-pl.rc

```diff
@@ -200,34 +201,34 @@ BEGIN
     EDITTEXT ID_EDT_LOG_DIR, 53, 72, 150, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_LOG_DIR, 208, 72, 25, 12
 
-    GROUPBOX "Script Timeout", 203, 6, 97, 235, 60
-    LTEXT "Preconnect script timeout:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 100, 10
-    LTEXT "Connect script timeout:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 90, 10
-    LTEXT "Disconnect script timeout:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 90, 10
+    GROUPBOX "Timeouty skryptów", 203, 6, 97, 235, 60
+    LTEXT "Przed połączeniem:", ID_TXT_PRECONNECT_TIMEOUT, 17, 110, 100, 10
+    LTEXT "Po połączeniu:", ID_TXT_CONNECT_TIMEOUT, 17, 125, 90, 10
+    LTEXT "Po rozłączeniu:", ID_TXT_DISCONNECT_TIMEOUT, 17, 140, 90, 10
     EDITTEXT ID_EDT_PRECONNECT_TIMEOUT, 103, 108, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
     EDITTEXT ID_EDT_CONNECT_TIMEOUT, 103, 123, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
     EDITTEXT ID_EDT_DISCONNECT_TIMEOUT, 103, 138, 20, 12, ES_AUTOHSCROLL|ES_NUMBER
 
-    GROUPBOX "Management interface", 204, 6, 163, 235, 30
-    LTEXT "Port offset:", 205, 17, 175, 75, 10
+    GROUPBOX "Interface zarządzania", 204, 6, 163, 235, 30
+    LTEXT "Offset portu:", 205, 17, 175, 75, 10
     EDITTEXT ID_EDT_MGMT_PORT, 103, 173, 30, 12, ES_AUTOHSCROLL
 
-    GROUPBOX "Config menu view", 206, 6, 198, 235, 30
-    AUTORADIOBUTTON "&Auto", ID_RB_BALLOON0, 28, 210, 50, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "&Flat", ID_RB_BALLOON1, 88, 210, 50, 10
-    AUTORADIOBUTTON "&Nested", ID_RB_BALLOON2, 148, 210, 50, 10
+    GROUPBOX "Wygląd menu połączeń", 206, 6, 198, 235, 30
```

Comment:
Could be `Wygląd menu konfiguracji` ?

Reply:
It could, but I decided to go that way because it seemed less ambiguous. After all we have:

* connections – the actual, running data links between client and server,
* OpenVPN GUI's own configuration – the one that is stored in registry,
* configurations – which are indeed configuration files, but in context of potential connections – more like “connection configs” (perhaps “konfiguracje połączeń”, but “Wygląd menu konfiguracji połączeń” is correct, but a mouthful nonetheless), “connection profiles” or an “addresses in address book” if you like.

This is not to negate your proposition, but to provide my rationale 😊👍